<launch>
  <env name="ROSCONSOLE_CONFIG_FILE" value="$(find dh_gripper)/rosconsole.conf"/>

  <arg name="gripper" default="AG-2E"/>
  <arg name="endpoint" default="192.168.1.29:8888"/>

  <!-- via USB -->
  <!--<param name="Connect_port" type="string" value="/dev/DH_hand"/>-->

  <!-- via TCP -->
  <node name="gripper_controller" pkg="dh_gripper" type="dh_controller" output="screen">
    <param name="gripper_model" value="$(arg gripper)"/>
    <param name="connect_port" value="$(arg endpoint)"/>
    <param name="data_timeout" value="0.05"/>
  </node>

</launch>
